### ISSUE TITLE
./script/docker_dev_setup.sh does not work

### ISSUE THREAD

**Issue by luvicsoura** [instructure/canvas-lms#1704]
### Steps to reproduce:

 1. run ./script/docker_dev_setup.sh
 2. login again, reboot do all that you can and keep stuck at
```
> We need to login again to apply that change.
OK to run 'exec sg docker -c ./script/docker_dev_setup.sh'? [y/n] 
```

### Expected behavior:
The script installs dory and sets up the development environment

### Actual behavior:
Answering `y` to the following prompt just ends the script

```
> We need to login again to apply that change.
OK to run 'exec sg docker -c ./script/docker_dev_setup.sh'? [y/n] 
```

### Additional notes:
Running Pop!_OS 20.04 (based on Ubuntu 20.04)
200+GiB free in disk

**Comment by juniorRubyist** [comment]
It's a shame that the quick script doesn't work as intended. It makes it hard for new contributors, like myself.

**Comment by inthebrilliantblue** [comment]
Ive been helping someone with setting up their own self hosted canvas install for the past year, and I dont think Ive ever been able to get it to work. I also had to create my own quick script to rapidly install canvas for testing using the production setup, which is also VERY out of date. Very sad indeed.

**Comment by rayfoss** [comment]
I have 20.04 it worked for me... I had to log out once, install ruby after the first failed run. I did restart after as well. The ICU dependency seems to be optional, but can be satisfied easily with icu_devtools.

related icu config deprecation https://github.com/LinearTapeFileSystem/ltfs/issues/153

Bigger issue is undoing the damage dory does... it's a virtual network adapter, bridge and vm configuration with a self hosted DNS server... seems way overkill for linux users who can run docker natively

**Comment by sijpkes** [comment]
> I have 20.04 it worked for me... I had to log out once, install ruby after the first failed run. I did restart after as well. The ICU dependency seems to be optional, but can be satisfied easily with libicu-dev.
> 
> related icu config deprecation [LinearTapeFileSystem/ltfs#153](https://github.com/LinearTapeFileSystem/ltfs/issues/153)
> 
> Bigger issue is undoing the damage dory does... it's a virtual network adapter, bridge and vm configuration with a self hosted DNS server... seems way overkill for linux/chromeOS users who can run docker natively

Yes, I'm having issues with the dory part too... do you have any tips on how to get this working?

**Comment by Ardena** [comment]
For Linux, we do not recommend Dory.  I believe our one or two Canvas engineers who work on Linux use Traefik instead.

**Comment by ccutrer** [comment]
Please use [the mailing list](https://groups.google.com/g/canvas-lms-users) for installation and operational issues.
